reliquery/tests/test_html.py

```diff
@@ -0,0 +1,50 @@
+import os
+import reliquery.relic as relic
+
+
+def test_list_html_file_when_add_html(tmpdir):
+    reliquery_dir = os.path.join(tmpdir, "reliquery")
+    os.makedirs(reliquery_dir)
+
+    rq = relic.Relic(name="element-bucket", relic_type="test")
```

Comment:
one small thing. when you make a relic to test, you wan to give it a test storage. otherwise its going to use the real storage:

like this: https://github.com/The-Dev-Effect/reliquery/blob/abd052a0c01c9ee60f74c28601dd290e4128a293/reliquery/tests/test_relic.py#L15



we may want to factor out something at some poitn to make test relics for us so we don't forget to to this.

Reply:
ok i see a version of using a fixture in another test file I'll try to make mine similar